### ISSUE TITLE
consolidate minimal, full and fullest record in tests

### ISSUE THREAD

**Issue by ppanero** [inveniosoftware/invenio-rdm-records#1254]
Should have only two fixtures `minimal` and `full`. The second one should be replaced with the more completed one `fullest`, some tests might need fixing since they will not take into account the full data.

**Comment by slint** [comment]
I was partially joking about the `fullest` record, just couldn't think of a better name 🤡 

So just to clarify we shouldn't name it like this, but rather update the the `full_record` to have more metadata.

**Comment by ppanero** [comment]
@slint I agree, I just didn't want to block the PR because updating the feature will break many other tests.

**Comment by fenekku** [comment]
Related to this: https://github.com/inveniosoftware/invenio-rdm-records/blob/master/tests/resources/serializers/conftest.py#L42 

Not a blocker for v13 I think but it would be nice to clear that all up. There should be fixtures for:

- minimal record dict input to REST API (we could even split it up in a JSON string and the resulting read dict (still input side) to be really thorough)
- full record dict input to REST API (equivalent of above)
- minimal dict `to_dict()` result representation
- same for full

There is often confusion between those that cause code to be written assuming less data than available (maybe other way too, but I've only seen cases of less data) and thus expensive operations are undertaken again. This was especially true in [serializers](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/tests/resources/serializers/__init__.py) at the time.
